<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FRFLoadDialog</class>
 <widget class="QDialog" name="FRFLoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Structural mesh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbStructuralMesh">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;#browseStructuralMesh&quot;&gt;(click to select file)&lt;/a&gt;</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbStateFileIdentifier">
        <property name="text">
         <string>State text file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbStateHistory">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;#browseStateHistory&quot;&gt;(click to select file)&lt;/a&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Excitations</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbExcite">
        <property name="text">
         <string>(n/a)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Eigenmodes</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lbEigenmodes">
        <property name="text">
         <string>(n/a)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>States/columns</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lbStates">
        <property name="text">
         <string>(n/a)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Rows</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="lbFrequencies">
        <property name="text">
         <string>(n/a)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Number of time steps</string>
        </property>
        <property name="buddy">
         <cstring>sbNumSteps</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="sbNumSteps">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbReferenceValues">
     <property name="title">
      <string>Reference Values</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Reference velocity</string>
        </property>
        <property name="buddy">
         <cstring>sbRefVelocity</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QScienceSpinBox" name="sbRefVelocity">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Refernce chord</string>
        </property>
        <property name="buddy">
         <cstring>sbRefChord</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QScienceSpinBox" name="sbRefChord">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbDynamicPressure">
        <property name="text">
         <string>Dynamic pressure</string>
        </property>
        <property name="buddy">
         <cstring>sbDynamicPressure</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QScienceSpinBox" name="sbDynamicPressure">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>State Mapping</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="cbDefaultMapping">
        <property name="text">
         <string>Use default mapping</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Assign state</string>
        </property>
        <property name="buddy">
         <cstring>sbStateIndex</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="sbStateIndex">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&gt; to eigenmode index</string>
        </property>
        <property name="buddy">
         <cstring>sbEigenmode</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="sbEigenmode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="specialValueText">
         <string>(unassigned)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&gt; to aerodynamic excitation</string>
        </property>
        <property name="buddy">
         <cstring>cbExciteTag</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="cbExciteTag"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbProceed">
       <property name="text">
        <string>&amp;Proceed...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbLoadSettings">
       <property name="text">
        <string>&amp;Load settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStoreSettings">
       <property name="text">
        <string>&amp;Store settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QScienceSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../qsciencespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>FRFLoadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDefaultMapping</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbStateIndex</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDefaultMapping</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbEigenmode</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDefaultMapping</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbExciteTag</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDefaultMapping</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDefaultMapping</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDefaultMapping</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
